RELEASING.md

```diff
@@ -166,10 +165,8 @@ Example: `<tag>v0.3.2</tag>`
 
 Update version-sensitive files in the root and documentation sources in the `docs` subdirectory:
 
-*   `docs/changelog.adoc`
-*   `docs/versions.adoc`
-*   `docs/upgrade.adoc`
-*   `docs/xsl/common.xsl`
+* `docs/changelog.md` - Add release section
```

Comment:
I think the release section should already be there at that point as contributors otherwise also can't add upgrade docs. Right now for example, we already have a section for `0.5.0`.
The release manager only needs to set the release date and make sure that the tested compatibility, as well as the linked GitHub milestone is correct.

Reply:
I think it is okay to have a release section for a unreleased (major) version, but not for any patch version, for example 0.4.2. I think this confuses users.

Reply:
How about then _Add / finalize release section_ or something like that?